ivy/functional/frontends/numpy/mathematical_functions/lcm_and_gcd.py

```diff
@@ -0,0 +1,43 @@
+import ivy
+
+
+def lcm(
+    x1,
+    x2,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="k",
+    dtype=None,
+    subok=True,
+):
+    if dtype:
+        x1 = ivy.astype(ivy.array(x1), ivy.as_ivy_dtype(dtype))
+        x2 = ivy.astype(ivy.array(x2), ivy.as_ivy_dtype(dtype))
+    ret = ivy.lcm(x1, x2, out=out)
```

Comment:
We don't have this function in ivy, which means that this function will need to be implemented compositionally by referring to the contribution guide here: [Frontend guide](https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html)